notebooks/cugraph_benchmarks/release.ipynb

```diff
@@ -0,0 +1,628 @@
+   "outputs": [],
+   "source": [
+    "try: \n",
+    "    import matplotlib\n",
+    "except ModuleNotFoundError:\n",
+    "    os.system('pip install matplotlib')\n",
+    "\n",
+    "import matplotlib.pyplot as plt; plt.rcdefaults()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the test data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Test File\n",
+    "data = {\n",
+    "    'preferentialAttachment' : './data/preferentialAttachment.mtx',\n",
+    "    'dblp'                   : './data/dblp-2010.mtx',\n",
+    "    'coPapersCiteseer'       : './data/coPapersCiteseer.mtx',\n",
+    "    'as-Skitter'             : './data/as-Skitter.mtx'\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read data\n",
+    "The data is read in once once and used for both cuGraph and NetworkX."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Data reader - the file format is MTX, so we will use the reader from SciPy\n",
+    "def read_data(datafile):\n",
+    "    print('Reading ' + str(datafile) + '...')\n",
+    "    M = mmread(datafile).asfptype()\n",
+    "\n",
+    "    _gdf = cudf.DataFrame()\n",
+    "    _gdf['src'] = M.row\n",
+    "    _gdf['dst'] = M.col\n",
+    "    \n",
+    "    return _gdf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create Graph functions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# NetworkX\n",
+    "def create_nx_graph(_df):\n",
+    "    _gnx = nx.from_pandas_edgelist(_df, source='src', target='dst', edge_attr=None, create_using=nx.DiGraph)\n",
+    "    return _gnx\n",
+    "\n",
+    "def create_nx_ugraph(_df):\n",
+    "    _gnx = nx.from_pandas_edgelist(_df, source='src', target='dst', edge_attr=None, create_using=nx.Graph)\n",
+    "    return _gnx\n",
+    "\n",
+    "\n",
+    "# cuGraph\n",
+    "def create_cu_graph(_df):\n",
+    "    _g = cugraph.DiGraph()\n",
+    "    _g.from_cudf_edgelist(_df, source='src', destination='dst', renumber=False)\n",
+    "    _ = _g.view_adj_list()\n",
+    "    return _g\n",
+    "\n",
+    "def create_cu_ugraph(_df):\n",
+    "    _g = cugraph.Graph()\n",
+    "    _g.from_cudf_edgelist(_df, source='src', destination='dst', renumber=False)\n",
+    "    _ = _g.view_adj_list()\n",
+    "    return _g"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### BFS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def nx_bfs(_df):\n",
+    "    t1 = time.time()\n",
+    "    _G = create_nx_ugraph(_df)\n",
+    "    _ = nx.bfs_edges(_G, 1)\n",
```

Comment:
Is assigning to the underscore var needed here?  Just wondering because it's not used elsewhere (eg. `perf.append([])` below) and doesn't read as nicely as without it.

Reply:
it is so that data is now outputted